### DISCUSSION TITLE
How do you manage your system settings?

### DISCUSSION THREAD

**Discussion by janlimpens** [YaLTeR/niri#476] [General]
What's holding me back on paperwm, is gnome's nice settings app and the gnome bar's quick settings. I have no interest at all in any *ricing* or mucking around in settings, I would want a productive system without much ado.
How do you guys go about this?

**Comment by YaLTeR** [comment]
Right, that's one disadvantage of WMs compared to full desktop environments, you need to add several of these components manually. Since building a WM/compositor is hard enough on its own, and building all of the components for it would need a whole team of people.

I'm not too experienced in the ricing part of the community (I myself have used GNOME for several years). I know people managed to get something close to GNOME's quick settings using layer-shell components ([example on niri](https://github.com/YaLTeR/niri/discussions/325#discussioncomment-9680351)), so you may have some luck copying their setup.

If you, understandably, don't want to deal with it, you can run a stock Waybar which gives you tray icons, then nm-applet will show an icon for connecting to Wi-Fi and VPN and opening network settings. Use pavucontrol to switch audio devices. Once #413 is in, you'll be able to use GUI tools to configure monitors.

You can also run GNOME Settings with `env XDG_CURRENT_DESKTOP=gnome gnome-control-center`. Keep in mind that a lot of settings will not work (they're *GNOME* Settings after all), but things like network, audio, printers should work just fine.

**Comment by janlimpens** [comment]
Great tips, thank you and congratulations to your great project!
